test/test-main.js

```diff
@@ -0,0 +1,12 @@
+var main = require("./main");
+
+exports["test main"] = function(assert) {
+  assert.pass("Unit test running!");
```

Comment:
Hey @hitesh96db have you looked any more into the testing framework?  

I took a cursory glance.  In particular, it appears to have an [unstable api](https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/test_assert).  This may be worth reviewing from time to time since a mature testing framework designed to test browser extensions could make life a lot easier.  However, right now it seems like it would be simpler to integrate it with the existing privly testing environment.